flux-accounting-guide.rst

```diff
@@ -162,3 +196,7 @@ bank
 .. _Slurm accounting: https://slurm.schedmd.com/accounting.html
 
 .. _multi-factor priority plugin: https://slurm.schedmd.com/priority_multifactor.html
+
+.. _jobtap: https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man7/flux-jobtap-plugins.html?highlight=jobtap
```

Comment:
This link highlights every instance of the word "jobtap" in the docs, was that intentional? If not, maybe you wanted just:

https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man7/flux-jobtap-plugins.html#flux-jobtap-plugins-7

Reply:
Oh I didn't even realize this! Thanks for pointing it out. I think there is only one instance of the word "jobtap" in the flux-accounting guide (with this PR), but I did not mean for it to highlight every potential future instance of the word. I'll change the link!

Reply:
Oh, I meant that it highlights every instance of the word `jobtap` in the target of the link. And there's a lot of instance of `jobtap` in `flux-jobtap-plugins(7)` 😆

Reply:
Ohhhh, oops. Lol. Sorry for misunderstanding! Yeah, in that case, I definitely did _not_ want to have the original link that I used, right? So the link that you provided is better.